examples/sandbox/src/App.js

```diff
@@ -26,6 +26,7 @@ export default function App() {
   return (
     <SearchProvider
       config={{
+        debug: true,
```

Comment:
Debug is default? Is that on purpose?

Reply:
@omegahm Just in the sandbox implementation. It is not on by default otherwise.